packages/services-store/src/actions/cloudAccountAuthAPI.ts

```diff
@@ -59,7 +59,9 @@ export function loadUserStatus(
 ): ThunkAction<Promise<UserStatus>, { cloudAccountAuthAPI: CloudAccountAuthAPIState }, void, CloudAccountAuthAPIActions> {
   return async dispatch => {
     const promiseStatus: Promise<UserStatus> =
-      authMethod === 'cookie' ? CloudAccountAuthAPI.checkCookie() : CloudAccountAuthAPI.checkAPI();
+      authMethod === 'cookie' || authMethod === 'personalization'
```

Comment:
I'm guessing 'cookie' is left here so any instances of the masthead won't break when we merge. How big of a concern is that? Is it conceivable we could update all instances of the masthead to use 'personalization' instead of 'cookie' once we merge this so we can remove all cookie-related code?

Reply:
@huntermacd That's correct! I left `cookie` here because it's being used on multiple applications and platforms so simply removing this value will have an adverse effect on those content ecosystems as we can't easily update the `authMethod` value across different adopters.

I think keeping it as it's, for now, should be fine and we can announce the use of the personalization method once we review this work with stakeholders and the seal of approval to merge these changes down.